packages/browser/rollup.config.js

```diff
@@ -8,7 +8,7 @@ const builds = [];
     isAddOn: false,
     jsVersion,
     licenseTitle: '@sentry/browser',
-    outputFileBase: `build/bundle${jsVersion === 'es6' ? '.es6' : ''}`,
+    outputFileBase: `${BUILD_DIR}/${BUNDLES_DIR}/bundle${jsVersion === 'es6' ? '.es6' : ''}`,
```

Comment:
You're making me realize this is going to be the same everywhere, and really belongs in the main rollup config.

https://github.com/getsentry/sentry-javascript/pull/4710

Reply:
Totally makes sense. Already approved the PR